templates/openwisp2/settings.py

```diff
@@ -293,6 +293,19 @@
     }
 }
 
+# HTML minification with django pipeline
+INSTALLED_APPS.append('pipeline')
+MIDDLEWARE.append('pipeline.middleware.MinifyHTMLMiddleware')
```

Comment:
Why not add them in `INSTALLED_APPS` and `MIDDLEWARE` list blocks above?

Reply:
Thought adding them in a separate block will make more sense. I will update it.